consensus/module.go

```diff
@@ -82,34 +82,104 @@ type consensusModule struct {
 type ConsensusDebugModule interface {
 	SetHeight(uint64)
 	SetRound(uint64)
-	SetStep(typesCons.HotstuffStep)
+	SetStep(uint64)
```

Comment:
1. That makes sense, I will update the definition to make it uint8 through the codebase.
2. Yes it requires additional refactoring, because currently `ConsensusPacemaker` interface is defined in shared, which requires to import `typesCons`, but currently it is not possible since this makes import cycle. I will add the `IMPROVE` comment.